Included caveats on GitHub reviews (#314)
palantir/policy-bot: README.md

disapproval. Without setting one of these fields, GitHub reviews that request
changes have no effect on the `policy-bot` status.

#### Interactions with GitHub Reviews

GitHub Reviews allow a user to dismiss the last review they left, causing it 

> to no longer count towards in rule evaluations. When this happens 

Reviewer: ```suggestion
to no longer count towards rule evaluations. When this happens 
```

> `policy-bot` will use the review before the dismissed review when evaluating 

Reviewer: ```suggestion
`policy-bot` will use a previous, non-dismissed review, if it exists, when evaluating
```
